dotnet-interactive/resources/dotnet-interactive.js

```diff
@@ -47,16 +49,24 @@
                                 getVariable: (variableName) => {
                                     return client.getVariable(kernelName, variableName);
                                 }
-                            }
+                            };
                         }
                     }
                 });
-        }
+        };
 
         global.interactive = client;
 
         client.loadKernels();
 
+        require([
```

Comment:
this can be done by setting the dependencies in the define above and then `lsp,cm` variable in the function defined.
Something like
```js
define(["dotnet-interactive/lsp", "dotnet-interactive/code-mirror-lsp"], function(lsp, cm) {
....
}
```

Reply:
I did it down here because I'm treating `dotnet-interactive.js` to be the root object/script and `lsp.js` (currently) requires the `global.interactive` variable to be set and I wasn't sure if it would break by trying to import lsp/code-mirror _before_ dotnet-interactive.js.

Reply:
I see, this makes sense!